changelogs/fragments/5592-redirect-remove-sap-modules.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - sapcar_extract - migrated to community.sap_libs.sapcar_extract
+  - sap_task_list_execute - migrated to community.sap_libs.sap_task_list_execute
+  - hana_query - migrated to community.sap_libs.sap_hdbsql
```

Comment:
Please look at https://github.com/felixfontein/community.general/blob/abecb17d0aefde99183c8444ca5b19264e85f8d1/changelogs/fragments/hetzner-migration-removal.yml and create something similar. You can cut out the 2.9/2.10 part, the collection now requires 2.11+.